packages/apm/src/integrations/tracing.ts

```diff
@@ -594,6 +668,9 @@ export class Tracing implements Integration {
       const hub = _getCurrentHub();
       if (hub) {
         const span = activeTransaction.child(spanContext);
+        if (options && typeof options.durationLimit === 'number') {
+          span.setData('durationLimit', options.durationLimit);
```

Comment:
Protect this with feature flag?

Maybe namespace into some debug object? -- will be easier to consume and to find code that add to the debug info later.